packages/shared/lib/services/sync/manager.service.ts

```diff
@@ -527,6 +527,7 @@ export class SyncManagerService {
             finishedAt: latestJob?.updated_at,
             nextScheduledSyncAt: schedule.nextDueDate,
             name: sync.name,
+            variant: sync.variant,
```

Comment:
unrelated to the other commits but I  realized I forgot to expose the variant in `getSyncStatus` response